add-tourpackage.php

```diff
@@ -322,6 +313,114 @@
                         </div>
                     </div>
                     <!-- destination selecttion model -->
+
+                    <!-- vehicle selection model -->
+                    <div class="modal fade" id="vehicle-selection-model" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+                        <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
```

Comment:
Add this `data-bs-backdrop="static"` attribute to this modal. Your model still have chance to hide after click another area of your web page. You can give static behavior to model in this way.

Refer this : https://getbootstrap.com/docs/5.2/components/modal/#static-backdrop

Reply:
done